src/main/java/duke/core/Parser.java

```diff
@@ -43,12 +43,11 @@ public Parser(String userInput) {
      * @return .
      * @throws DukeException .
      */
-    public String[] parseDeletePatientTask() throws DukeException {
-        String[] formattedInput = new String[2];
+    public String parseDeletePatientTask() throws DukeException {
+        String formattedInput;
         try {
-            String[] parsedCommand = userInput.split("\\s+", 4);
-            formattedInput[0] = parsedCommand[2];
-            formattedInput[1] = parsedCommand[3];
+            String[] parsedCommand = userInput.split("\\s+", 3);
+            formattedInput = parsedCommand[2];
```

Comment:
Same goes to Parser. Making changes to part not belong to you will cause others to have conflict. Discard the changes pls.

Reply:
noted